client/package.json

```diff
@@ -16,7 +16,13 @@
     "redux-thunk": "^2.2.0"
   },
   "devDependencies": {
-    "react-scripts": "1.1.4"
+    "chromedriver": "^2.42.1",
+    "cucumber": "^5.0.2",
+    "cucumber-pretty": "^1.4.4",
+    "nightwatch": "^0.9.21",
+    "nightwatch-cucumber": "^9.1.3",
+    "react-scripts": "1.1.4",
```

Comment:
could be `^1.1.4` (it has been changed in `api-platform/api-platform` IIRC)

Reply:
Fixed in #56